server/config/subject_page.proto

```diff
@@ -164,9 +164,36 @@ message ScatterTileSpec {
 }
 
 message BarTileSpec {
+  enum SortType {
+    SORT_ASCENDING = 1;
+    SORT_DESCENDING = 2;
+    SORT_ASCENDING_POPULATION = 3;
+    SORT_DESCENDING_POPULATION = 4;
+  }
```

Comment:
@pradh @dwnoble I'm less familiar with how protobufs and MessageToJson() works. This enum is intended to match these lines: https://github.com/datacommonsorg/website/blob/59601b22c428665893ff8b78e88da0551e597f24/static/js/chart/types.ts#L170-L178. Is this the right approach?

Reply:
I think the JS code will see the sort field value as a string "SORT_DESCENDING", that you might then need to map to "descending".

I would recommend just using identically named enum names, so that's just a lower() op?

Reply:
Thanks! Swapped to identically named enum names, and added a typecasting function